packages/flutter/lib/src/foundation/stack_frame.dart

```diff
@@ -78,6 +78,8 @@ class StackFrame {
         // On the Web in non-debug builds the stack trace includes the exception
         // message that precedes the stack trace itself. fromStackTraceLine will
         // return null in that case. We will skip it here.
+        // TODO(polina-c): if one of lines was parsed to null, the entire stack trace
```

Comment:
I believe this TODO format will be blocked by a lint. Flutter style guide requires a link to a github issue attached to every TODO.

Reply:
ha-ha, yes
I created issue, but forgot to link. Thanks.